modules/install-sno-installing-with-usb-media.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="installing-with-usb-media_{context}"]
+
+= Installing with USB media
+
+Installing with USB media involves creating a bootable USB drive with the discovery ISO on the administration node. Booting the server with the USB drive prepares the node to install singe node OpenShift (SNO) via the Assisted Installer (AI).
+
+.Procedure
+
+. On the administration node, insert a USB drive into a USB port.
+
+. Create a bootable USB drive.
++
+[source,terminal]
+----
+# dd if=<path-to-iso> of=<path/to/usb> status=progress
+----
++
+For example:
++
+[source,terminal]
+----
+# dd if=discovery_image_sno.iso of=/dev/sdb status=progress
+----
++
+Once complete, the USB drive is ready for installation on the server.
+
+. On the server, insert the USB drive into a USB port.
+
+. Reboot the server and enter the BIOS settings upon reboot.
+
+. Change boot drive order to make the USB drive boot first.
+
+. Save and exit the BIOS settings. The server will boot with the discovery image.
+
+. On the administration node, return to the browser and refresh the page. If necessary, reload the link:https://cloud.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer] page and select the cluster name.
+
+. Click *Next* until you reach step 3.
```

Comment:
I suppose that you or someone else will be notified if a new step is introduced.  If you happen to know the title of the page, I'd specify that instead of the number.

Reply:
I can't seem to log in to see the tab names. This is a developer preview. So it won't be widely circulated before we have the opportunity to review it again.